datasets/covid19_italy/national_trends/pipeline.yaml

```diff
@@ -0,0 +1,161 @@
+        resources:
+          request_memory: "4G"
+          request_cpu: "1"
+
+    - operator: "GoogleCloudStorageToBigQueryOperator"
+      description: "Task to load CSV data to a BigQuery table"
+
+      args:
+        task_id: "load_covid19_italy_national_trends_to_bq"
+
+        # The GCS bucket where the CSV file is located in.
+        bucket: "{{ var.json.shared.composer_bucket }}"
+
+        # The GCS object path for the CSV file
+        source_objects: ["data/covid19_italy/national_trends/data_output.csv"]
+        source_format: "CSV"
+        destination_project_dataset_table: "covid19_italy.national_trends"
+
+        # Use this if your CSV file contains a header row
+        skip_leading_rows: 1
+
+        # How to write data to the table: overwrite, append, or write if empty
+        # See https://cloud.google.com/bigquery/docs/reference/auditlogs/rest/Shared.Types/WriteDisposition
+        write_disposition: "WRITE_TRUNCATE"
+
+        # The BigQuery table schema based on the CSV file. For more info, see
+        # https://cloud.google.com/bigquery/docs/schemas.
+        # Always use snake_case and lowercase for column names, and be explicit,
+        # i.e. specify modes for all columns.
+        # types: "INTEGER", "TIMESTAMP", "STRING"
+        schema_fields:
+          - name: "date"
+            type: "TIMESTAMP"
+            mode: "NULLABLE"
+          - name: "country"
+            type: "STRING"
+            mode: "NULLABLE"
+          - name: "hospitalized_patients_symptoms"
+            type: "INTEGER"
+            mode: "NULLABLE"
+          - name: "hospitalized_patients_intensive_care"
+            type: "INTEGER"
+            mode: "NULLABLE"
+          - name: "total_hospitalized_patients"
+            type: "INTEGER"
+            mode: "NULLABLE"
+          - name: "home_confinement_cases"
+            type: "INTEGER"
+            mode: "NULLABLE"
+          - name: "total_current_confirmed_cases"
+            type: "INTEGER"
+            mode: "NULLABLE"
+          - name: "new_current_confirmed_cases"
+            type: "INTEGER"
+            mode: "NULLABLE"
+          - name: "new_total_confirmed_cases"
+            type: "INTEGER"
+            mode: "NULLABLE"
+          - name: "recovered"
+            type: "INTEGER"
+            mode: "NULLABLE"
+          - name: "deaths"
+            type: "INTEGER"
+            mode: "NULLABLE"
+          - name: "total_confirmed_cases"
+            type: "INTEGER"
+            mode: "NULLABLE"
+          - name: "tests_performed"
+            type: "INTEGER"
+            mode: "NULLABLE"
+          - name: "note"
+            type: "STRING"
+            mode: "NULLABLE"
+
+  graph_paths:
+    - "covid19_italy_national_trends_transform_csv >> load_covid19_italy_national_trends_to_bq"
+
+
+
+
+
```

Comment:
Remove these lines (keep only one blank line at the end of every file)

Reply:
Done.